project/PekkoDependency.scala

```diff
@@ -53,7 +53,7 @@ object PekkoDependency {
   }
 
   // Default version updated only when needed, https://pekko.apache.org/docs/pekko/current/project/downstream-upgrade-strategy.html
-  val minimumExpectedPekkoVersion = "0.0.0+26720-01379c41-SNAPSHOT"
+  val minimumExpectedPekkoVersion = "1.0.0-rc1"
```

Comment:
Is it possible to capatlize the RC here, i.e. "1.0.0-RC1". Even if it doesn't make a difference, I have never seen a lowercase convention when it comes to OS Scala RC's.

Reply:
the files are named with rc1 - so this is more correct

please see https://repository.apache.org/content/groups/staging/org/apache/pekko/pekko-actor_3/1.0.0-rc1/

Reply:
https://lists.apache.org/thread/4hkt05tsqc82jr1oqsw39or0pwxk2ym4